static-docs/docs/index.md

```diff
@@ -146,6 +146,12 @@ Install the library using make
 make install
 ```
 
+Run `ldconfig` to update library cache
+
+```text
+sudo ldconfig
+```
+
```

Comment:
ldconfig is a development environment specific so I would rather not include it in the build docs. For example when I'm testing, I build and install to a build dir under the current directory and run the tests with:

LD_LIBRARY_PATH=$(pwd)/build ./gpio-test